src/components/project/project.service.ts

```diff
@@ -554,6 +555,20 @@ export class ProjectService {
       pinned: !!result.pinnedRel,
       scope: result.memberRoles.flat().map(rolesForScope('project')),
     };
+
+    if (changeId) {
```

Comment:
I was planning to return only CR pending properties and other fields should be undefined or null but several fields are non-nullable in Project DTO. So I returned origin active properties if they are not CR requested.

Reply:
Yeah that's what I was thinking anyways 👍